src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1414,43 +1403,36 @@ pre.rust {
 
 .theme-picker {
 	position: absolute;
-	left: -34px;
-	top: 9px;
+	left: -38px;
+	top: 4px;
 }
 
 .theme-picker button {
 	outline: none;
 }
 
 #settings-menu, #help-button {
-	position: absolute;
-	top: 10px;
-}
-
-#settings-menu {
-	right: 0;
+	margin-left: 4px;
 	outline: none;
 }
 
+#theme-picker, #copy-path {
+	height: 34px;
+}
```

Comment:
These can be `height: 100%`, right?

Reply:
None of them are in the container. Also, theme-picker will be shortly removed so I didn't want to refactor more than necessary by including it into the container. As for copy-path, it's on a completely different element.